src/main/java/ca/corefacility/bioinformatics/irida/service/ProjectSubscriptionService.java

```diff
@@ -0,0 +1,24 @@
+package ca.corefacility.bioinformatics.irida.service;
+
+import ca.corefacility.bioinformatics.irida.model.subscription.ProjectSubscription;
+import ca.corefacility.bioinformatics.irida.model.user.User;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Sort;
+
+/**
+ * A specialized service layer for project subscriptions.
+ */
+public interface ProjectSubscriptionService extends CRUDService<Long, ProjectSubscription> {
```

Comment:
Can you fix the formatting of this file, it should be indented with tabs and not spaces. Might just need to update your ide to set tabs as default for .java files.

Reply:
Please see 242aeb0522f0fe6211596050c7f3a2dcc6ef7b32.